apps/console/src/features/applications/components/settings/attribute-management/attribute-list-item.tsx

```diff
@@ -156,7 +190,10 @@ export const AttributeListItem: FunctionComponent<AttributeListItemPropInterface
     }, [ claimMappingOn ]);
 
     return (
-        <Table.Row data-testid={ testId }>
+        <Table.Row data-testid={ testId + "" }>
```

Comment:
Any reason to add "" to data-testid?

Reply:
No I guess its a mistake. Will resolve it.

Reply:
resolved with https://github.com/wso2/identity-apps/pull/3528/commits/acf93bbed6a21c13a9d22d84cc8294ea664ad8c4